ui/src/app/about/components/about-more/about-details.component.html

```diff
@@ -50,6 +50,15 @@ <h2>Enabled Features</h2>
             <span class="fa fa-close"></span>
           </strong>
         </div>
+        <div class="line">
+          <span>Wavefront: </span>
+          <strong *ngIf="dataflowVersionInfo.featureInfo.wavefrontEnabled" id="wavefrontEnabled">
```

Comment:
Does this mean we are going to have a feature flag `wavefrontEnabled` in SCDF (backend)?

Reply:
Yes, see https://github.com/spring-cloud/spring-cloud-dataflow/pull/4093